content/05.limitations.md

```diff
@@ -0,0 +1,17 @@
+## Limitations of Study
+
+<!-- We suggest that you divide your limitations section into three steps: (1) identify the study limitations; (2) explain how they impact your study in detail; and (3) propose a direction for future studies and present alternatives.-->
+
+
+This study has some potential limitations.
+The brain tumor samples were collected over decades, RNA samples were prepared using more than one type of library method (stranded or poly-A, **Figure {@fig:S7}A**), and were sequenced at multiple centers.
+While we noted a strong library preparation batch effect (**Figure {@fig:S7}B**) and possible sequencing center batch effect (**Figure {@fig:S7}C**), we also found a very unbalanced distribution of cancer groups with stranded or poly-A RNA-Seq.
+We did not perform batch correction because removal of batch effects across unbalanced groups may induce false differences between groups in some cases [@doi:10.1093/biostatistics/kxv027; @doi:10.1016/j.tibtech.2017.02.012]. 
+Batch correction strategy is highly dependent on an analyst’s goals [@doi:10.1016/j.tibtech.2017.02.012], therefore, we made expression matrices available by library type and used only stranded expression matrices for transcriptomic analyses in (**Figure {@fig:4}**) and (**Figure {@fig:5}**).
+A second potential limitation is that performing analyses with all samples, rather than samples with high tumor purity might result in loss of information, such as subclonal variants or low-level oncogenic pathway expression.
+We observe potential cancer-specific correlations in tumor purity (e.g. lower purity in SEGA, PXA, teratoma (**Figure {@fig:S3}A**).
+To confirm our analyses support true tumor biology, we performed the same transcriptomic analyses using only samples at or above median tumor purity within their cancer group (**Figure {@fig:S7}D-I**).
+Overall, the thresholded tumor purity results were broadly consistent with the results utilizing the full cohort of samples.
+Finally, this initial PBTA cohort is skewed towards samples collected at diagnosis from one tumor section/punch.
+In patients with duplicate sequenced samples per tumor event, we randomly selected one independent specimen for analysis and did not find notable differences if the specimen changed over time (i.e. with a new data release).
+We were unable to reliably perform systematic intratumoral and/or longitudinal analyses in this cohort, though we expect nearly 100 paired longitudinal tumors from the ([NIH X01 CA267587-01 pediatric brain tumor cohort](https://commonfund.nih.gov/kidsfirst/2021X01projects#FY21_Resnick)) to be released through the [Open Pediatric Cancer (OpenPedCan) project](https://github.com/PediatricOpenTargets/OpenPedCan-analysis)).
```

Comment:
I think I would make this the 3rd sentence maybe? After saying "the samples come from many places" we can update this to say, "As a consequence, we were unable to..."

What do you think?

Reply:
Since this is now the last last sentence of the paper, Maybe better to swap lines 15 and 16 instead of 16 and 17. This way, it directs readers to the next phase?

Reply:
Reworded in https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/435/commits/9923b5ff3043019c02d4db07a2aa9e8c78951c14